### PR TITLE
Add UTM pip package

### PR DESCRIPTION
**Pull request by anassinator**
https://pypi.python.org/pypi/utm

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2164,6 +2164,19 @@ python-usb:
     raring: [python-usb]
     saucy: [python-usb]
     trusty: [python-usb]
+python-utm:
```
Comment (dirk-thomas) [inline rosdep/python.yaml]: The key should include `pip` to have an upgrade path once this package is being packaged by the platforms.

Comment (anassinator) [inline rosdep/python.yaml]: Fixed. Need I squash into one commit?

Comment (dirk-thomas) [inline rosdep/python.yaml]: Thanks, no squashing is not necessary.

**Comment by dirk-thomas** [comment]
+1

**Pull request merged**